include/aws/common/private/lookup3.c

```diff
@@ -328,6 +336,8 @@ static uint32_t hashlittle( const void *key, size_t length, uint32_t initval)
      * noticably faster for short strings (like English words).
      */
 #ifndef VALGRIND
+#pragma CPROVER check push
+#pragma CPROVER check disable "pointer"
```

Comment:
The following block of code up to `#pragma CPROVER check pop` may read beyond the end of the string, as documented in the code. However, we consider these accesses as benign.

Reply:
I would suggest to include a comment to this effect in the source code. Nobody is going to look at the PR comments at any future date.